src/ecs/rendering/systems.rs

```diff
@@ -38,18 +44,54 @@ where
         Fetch<'a, Factory>,
         FetchMut<'a, EventChannel<Event>>,
         FetchMut<'a, WindowMessages>,
+        FetchMut<'a, ScreenDimensions>,
         <P as PipelineData<'a>>::Data,
     );
 
-    fn run(&mut self, (factory, mut event_handler, mut window_messages, data): Self::SystemData) {
+    fn run(
+        &mut self,
+        (
+            factory,
```

Comment:
Is this standard formatting?

Reply:
Probably not, but rustfmt-nightly also seems to refuse to touch it, I'm not quite sure why.

Reply:
Odd, but okay.